client/gutenberg/editor/style.scss

```diff
@@ -226,14 +228,16 @@ $gutenberg-theme-toggle: #11a0d2;
 .edit-post-options-modal {
 
 	// UNSET CALYPSO DEFAULT STYLES
-	input[type='text'],
-	input[type='search'] {
-		width: auto;
-	}
-	.components-text-control__input input[type='text'] {
+				
+input[type='text'],
```

Comment:
It seems that Prettier is ignoring the `/** @format */` directive, so can you indent this lines according to [Calupso CSS/Sass Coding guidelines](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md)? They are based on the [WordPress CSS Coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/), so this block should be indented with one tab:

```css
	input[type='text'],
	input[type='search'] {
		width: auto;

		&.components-text-control__input {
			width: 100%;
		}
	}
```

instead of

```css
input[type='text'],
input[type='search'] {
	width: auto;

	&.components-text-control__input {
		width: 100%;
	}

}
```